common/autoware_universe_utils/include/autoware/universe_utils/system/time_keeper.hpp

```diff
@@ -61,9 +61,10 @@ class ProcessingTimeNode : public std::enable_shared_from_this<ProcessingTimeNod
   /**
    * @brief Construct a ProcessingTimeTree message from the node and its children
    *
-   * @return tier4_debug_msgs::msg::ProcessingTimeTree Constructed ProcessingTimeTree message
+   * @return autoware_internal_debug_msgs::msg::ProcessingTimeTree Constructed ProcessingTimeTree
+   * message
    */
-  tier4_debug_msgs::msg::ProcessingTimeTree to_msg() const;
+  autoware_internal_debug_msgs::msg::ProcessingTimeTree to_msg() const;
```

Comment:
This message is now still in the tier4_debug_msgs.

Reply:
They are now ported into autoware_internal_msgs: https://github.com/autowarefoundation/autoware_internal_msgs/pull/41/files